app/models/user.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  PASSWORD_COMPLEXITY_REGEXP = /(?=.*?\w)(?=.*?[0-9])(?=.*?[#?!@$%^&*-])/.freeze
+
+  devise :database_authenticatable, :rememberable, :validatable, :recoverable, :confirmable
+
+  validates :email, :first_name, :last_name, presence: true
```

Comment:
Devise already includes validation for email presence, this second one will result in duplicate error messages regarding email presence